core/src/main/java/com/linecorp/armeria/server/http/dynamic/Converters.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.http.dynamic;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * The containing annotation type for {@link Converter}.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface Converters {
```

Comment:
After reviewing the documentations, I found that we need container annotation to repeat the same annotations - without it, a compilation occurs when a user tries to repeat the annotation. It is why I decided that `Converters` annotation is necessary.

- [Repeating Annotations (Oracle official Java guide)](http://docs.oracle.com/javase/tutorial/java/annotations/repeating.html)
- [Java 8 Repeating Annotation Explained in 5 minutes](https://blog.idrsolutions.com/2015/03/java-8-repeating-annotation-explained-in-5-minutes/)

However, after thinking carefully about your comment, I agree that `Converters` don't need to go public. Then, how about make this annotation package-private?